db/seeds.rb

```diff
@@ -39,15 +43,28 @@
   info_url: 'http://www.aw.org',
   donation_url: 'https://aw.myschoolapp.com/Page/Support/Make-a-Gift?siteId=693&ssl=1'
 },{
-  name: 'Girls Who Code',
-  description: 'Mission: Launched in Spring 2012, Girls Who Code is a national nonprofit organization working to close
-                the gender gap in the technology and engineering sectors. With support from public and private partners,
-                Girls Who Code works to educate, inspire, and equip high school girls with the skills and resources to
-                pursue opportunities in computing fields.',
-  reason_to_donate: "I’d like to support young girls in tech",
-  slug: 'girls-who-code',
-  info_url: 'https://girlswhocode.com/',
-  donation_url: 'https://girlswhocode.com/'
+  name: 'Black Girls Code',
+  description: "Black Girls Code's mission is to increase the number of women of color in the digital space by empowering girls of color
+                ages 7 to 17 to become innovators in STEM fields, leaders in their communities, and builders of their
+                own futures through exposure to computer science and technology.",
+  reason_to_donate: 'I feel like diversity in tech is important',
+  slug: 'black-girls-code',
+  info_url: 'http://www.blackgirlscode.com',
+  donation_url: 'http://www.blackgirlscode.com'
+},{
+  name: 'Southern Poverty Law Center',
+  description: 'The Southern Poverty Law Center is dedicated to fighting hate and bigotry and to seeking justice for the most vulnerable members of our society. Using litigation, education, and other forms of advocacy, the Center works toward the day when the ideals of equal justice and equal opportunity will be a reality.',
+  reason_to_donate: 'I believe in fighting bigotry',
+  slug: 'southern-poverty-law-center',
+  info_url: 'http://www.splcenter.org/',
+  donation_url: 'https://donate.splcenter.org/sslpage.aspx?pid=463'
+},{
+  name: 'Center for Reproductive Rights',
+  description: 'For 20 years, the Center for Reproductive Rights has used the law to advance reproductive freedom as a fundamental human right that all governments are legally obligated to protect, respect, and fulfill.',
```

Comment:
Line is too long. [221/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.